src/_listByline.scss

```diff
@@ -101,7 +102,7 @@
       color: $fd-list-byline-section-text-color;
       padding-right: $fd-list-byline-elements-spacing;
       line-height: $fd-list-byline-text-line-height;
-      align-self: flex-end;
+      align-self: flex-start;
```

Comment:
why was this changed?

Reply:
i think some of the css present in Ngx is un-necessary and could have been avoided. i am not making any change for that because it is already working and approved code.
This change is present in https://github.com/SAP/fundamental-ngx/blob/main/libs/core/src/lib/list/list.component.scss, so i have moved it to styles.

Reply:
reverted back now. i do not see any issue using 'flex-end', examples are working fine. so no need to bring Ngx change for this.